cumulusci.yml

```
@@ -608,29 +582,41 @@ flows:
             4:
                 task: publish_learner_portal
 
-    run_unmanaged_unit_tests:
+    qa_org_2gp_ftests:
+        group: "Org Setup"
+        description: Set up an org and run functional tests as a QA environment using a second-generation package
+        steps:
+            1:
+                flow: qa_org_2gp
+            2:
+                flow: run_functional_tests
+
+    run_functional_tests:
         # description: Configure an org for use as a dev org after package metadata is deployed.
         steps:
             1:
-                task: deploy_unpackaged_test_metadata
-                when: "'hed' not in org_config.installed_packages"
+                task: deploy_functional_test_utilities
+                when: "'hed' in org_config.installed_packages"
             2:
-                task: deploy_unpackaged_primary_affiliations_config
-                when: "'hed' not in org_config.installed_packages"
+                task: deploy_functional_tests
+                when: "'hed' in org_config.installed_packages"
             3:
-                task: execute_primary_affiliation_mapping_apex
-                when: "'hed' not in org_config.installed_packages"
-            4:
-                task: deploy_unpackaged_relationship_autocreate_config
+                # execute tests in unmanaged package
+                task: run_tests
+                options:
+                    test_name_match: '%\_FTST'
```

Need to add a blank namespace to make `run_tests` task to run these ftests
```suggestion
                    test_name_match: '%\_FTST'
                    namespace: ''
```